src/api/lib/db/queries/updateOne.js

```diff
@@ -27,7 +27,7 @@ export default (
     }
 
     let shouldTriggerNewVersion = false;
-    if (version && (yield hasChange(parameters)) === true) {
+    if (version && (await hasChange(parameters)) === true) {
```

Comment:
I don't understand if `hasChange` mean `hasChanged` or `hasChanges` or just `changed`... (not the aim of the PR... I know)